netjsonconfig/backends/openvpn/schema.py

```diff
@@ -190,6 +190,17 @@
                     "pattern": "^(\\S*)$",
                     "propertyOrder": 17,
                 },
+                "tls_auth": {
+                    "title": "TLS Auth",
+                    "description": (
+                        "Adds an additional layer of HMAC authentication on top of "
+                        "the TLS control channel to mitigate DoS attacks and "
+                        "attacks on the TLS stack"
+                    ),
+                    "type": "string",
+                    "format": "textarea",
+                    "propertyOrder": 18,
```

Comment:
It has same `propertyOrder` as `crl_verify`. 

https://github.com/openwisp/netjsonconfig/blob/f2e692cee7d3a26b59c3a8143ddfcbc97816445c/netjsonconfig/backends/openvpn/schema.py#L622-L628